pyproject.toml

```diff
@@ -136,6 +136,7 @@ exclude_also = ["if __name__ == .__main__.:", "if TYPE_CHECKING:", "if ._pyodide
 
 [tool.ruff]
 fix = true
+exclude = ["holoviews/tests/"]
```

Comment:
This is not correct.

Reply:
It's not? It seems to have helped with the CI failures now, except the `doc/generate_modules.py` file which is not in the tests dir.

Reply:
You are excluding _all_ linting in this path. 

You have two options: either fix the code the CI complains about or add the path to the `[tool.ruff.lint.per-file-ignores]` section. (I would prefer the first).

Reply:
OK. The first option means updating the docstrings in the tests dir, which was something I deliberately ignored since they're not part of the public API. Maybe the second option is best here.